humagne.yaml

```diff
@@ -553,7 +553,7 @@ packages:
       - cp2k+mpi+plumed
         ^fftw+mpi~openmp+fma simd=sse2,avx,avx2,avx512
         ^plumed+mpi+gsl
-      - molpro@2015.1+mpi
+      - molpro@2019.2+mpi
```

Comment:
We should keep the existing one and add the later in addition. Otherwhy spack-packagelist will not correspond to the installed softwares (there are not auto uninstalls)

Reply:
Fine by me. It also helps with the license issues, since the 2019 version expires in 1 year, whereas the 2015 does not expire at it may very well be the long term version.